TileStache/Goodies/VecTiles/transform.py

```diff
@@ -3185,3 +3185,14 @@ def elevation_to_meters(shape, props, fid, zoom):
 
     props['elevation'] = _to_float_meters(elevation)
     return shape, props, fid
+
+
+def add_is_bicycle_route(shape, props, fid, zoom):
+    """
+    If the props contain a bicycle_network tag or cycleway, it should
+    have an is_bicycle_route boolean.
+    """
+    props.pop('is_bicycle_route', None)
+    if 'bicycle_network' in props or 'cycleway' in props:
```

Comment:
Yeah, agree we should include the variants.

Looks like sometimes the variants can add up to something that should end up in `cycleway` (e.g.: if `cycleway:left=yes` and `cycleway:right=yes`, then really it's just `cycleway=yes`). We can tackle that in a later issue in the next release.

Similarly, seems like the `oneway:bicycle=no` thing is to counteract when the parent road is `oneway:yes` (to mean the cycleway is bidirectional and both directions probably on one side of the road).